src/irr/asset/CPLYMeshFileLoader.cpp

```diff
@@ -150,7 +145,7 @@ asset::SAssetBundle CPLYMeshFileLoader::loadAsset(io::IReadFile* _file, const as
 				else
 				{
 					// get element
-					std::unique_ptr<SPLYElement> el = std::move(ctx.ElementList[ctx.ElementList.size()-1]);
+					core::smart_refctd_ptr<SPLYElement> el = std::move(ctx.ElementList[ctx.ElementList.size()-1]);
```

Comment:
you can use unique_ptr on non-IReferenceCounted objects, in fact `unique_ptr` is great!